### PR TITLE
Feature/examples datepicker

### PR DESCRIPTION
**Pull request by lievenvg**
## PR Checklist

This PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [Contributing guidelines](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] A [changelog entry](https://github.com/digipolisantwerp/acpaas-ui_angular/blob/master/guidelines/CHANGELOG.md) was added

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/),
 and this project adheres to [Semantic Versioning](https://semver.org/).
 
 ## [Unreleased] - yyyy-mm-dd
+### Added
+- `datepicker` Added documentation.
```
Comment (TriangleJuice) [inline CHANGELOG.md]: This should not be added to the changelog

File: packages/forms/examples/forms/pages/datepicker/datepicker.page.html [hunk 2]
```diff
@@ -1 +1,34 @@
...
+	<aui-code-snippet [codeSnippet]="datepickerExampleTypescript"></aui-code-snippet>
+</div>
+
+<div class="u-margin-bottom">
+	<aui-code-snippet [codeSnippet]="datepickerExampleHTML"></aui-code-snippet>
+</div>
+
+<div class="u-margin-bottom-lg">
+	<form [formGroup]="dateForm">
+		<div class="a-input has-icon-right u-margin-bottom-xs col-md-6 col-xs-12" [ngClass]="{'has-error': dateForm.controls.inputDate.dirty && dateForm.controls.inputDate.invalid}">
```
Comment (TriangleJuice) [inline packages/forms/examples/forms/pages/datepicker/datepicker.page.html]: Please delete the flexbox classes.

File: packages/forms/examples/forms/pages/datepicker/datepicker.page.ts [hunk 3]
```diff
@@ -1,7 +1,70 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+
+import { DateRange } from '@acpaas-ui/js-date-utils';
 
 @Component({
-	templateUrl: './datepicker.page.html',
+  templateUrl: './datepicker.page.html',
```
Comment (TriangleJuice) [inline packages/forms/examples/forms/pages/datepicker/datepicker.page.ts]: Fix indenting please. On a sidenote, I don't get why your code editor is doing this by default, maybe you should check your settings.

File: packages/forms/examples/forms/pages/datepicker/datepicker.page.ts [hunk 4]
```diff
@@ -1,7 +1,70 @@
...
+
+export class AppModule {};`;
+	public datepickerExampleTypescript = `import { FormBuilder, FormGroup } from '@angular/forms';
+
+import { DateRange } from '@acpaas-ui/js-date-utils';
+
+constructor(private fb: FormBuilder) { }
+
+public dateRange: DateRange = [
+	5, 6,
```
Comment (TriangleJuice) [inline packages/forms/examples/forms/pages/datepicker/datepicker.page.ts]: I would put this on one line

File: packages/forms/src/lib/datepicker/README.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
+# @acpaas-ui/ngx-components/forms
+The package creates a custom input component allowing the user to select a date either by input or by picking one in the calendar flyout.
+
+## Dependencies
```
Comment (TriangleJuice) [inline packages/forms/src/lib/datepicker/README.md]: I'm not sure if we should list these as dependencies (with the exception of `@acpaas-ui/js-date-utils`).

File: packages/forms/src/lib/datepicker/README.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+In the example below we use a reactive form.
+```typescript
+import { FormBuilder, FormGroup } from '@angular/forms';
+
+import { DateRange } from '@acpaas-ui/js-date-utils';
+
+constructor(private fb: FormBuilder) { }
+
+public dateRange: DateRange = [
+  5, 6,
```
Comment (TriangleJuice) [inline packages/forms/src/lib/datepicker/README.md]: Same comment as above.

File: packages/forms/src/lib/datepicker/README.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+| `@Input() autocomplete: string;` | `'off'` |  Turn the browsers autocompletion on or off. |
+| `@Input() range: DateRange;` | - | A range of dates or weekdays to disable (see the [@acpaas-ui/js-date-utils](https://github.com/digipolisantwerp/acpaas-ui_js/blob/master/packages/date-utils/README.md) package for more info on date ranges). |
+
+#### Example
+
+You can provide custom month and weekday labels as wel as error labels via the `forChild` method.
+```typescript
+import { DatepickerModule } from '@acpaas-ui/ngx-components/forms';
+
+@NgModule({
```
Comment (TriangleJuice) [inline packages/forms/src/lib/datepicker/README.md]: Fix indenting please (in the whole README file)

File: packages/forms/src/lib/datepicker/datepicker.module.ts [hunk 8]
```diff
@@ -11,9 +11,9 @@ import {
 	CALENDAR_DEFAULT_MONTH_LABELS
 } from '@acpaas-ui/ngx-components/calendar';
 
-import { MaskModule } from '../mask';
+import { MaskModule } from '../mask/mask.module';
```
Comment (TriangleJuice) [inline packages/forms/src/lib/datepicker/datepicker.module.ts]: Have you tried `import { MaskModule } from '@acpaas-ui/ngx-components/mask'`?

File: CHANGELOG.md (hunk 1 above)
Comment (lievenvg) [reply CHANGELOG.md]: Is this not consistent with the previous changelog update, where added documentation is mentioned?

File: packages/forms/src/lib/datepicker/README.md (hunk 5 above)
Comment (lievenvg) [reply packages/forms/src/lib/datepicker/README.md]: It's in line with the other examples, but we can always review it.

File: packages/forms/src/lib/datepicker/datepicker.module.ts (hunk 8 above)
Comment (lievenvg) [reply packages/forms/src/lib/datepicker/datepicker.module.ts]: I have now... 😉

File: packages/forms/src/lib/datepicker/README.md (hunk 5 above)
Comment (TriangleJuice) [reply packages/forms/src/lib/datepicker/README.md]: No, it's only in line with the one's we overlooked, i.e.: `table` and `flyout button`. The packages are in the same repository, so shouldn't be listed as dependency.

File: CHANGELOG.md (hunk 1 above)
Comment (TriangleJuice) [reply CHANGELOG.md]: My bad, I thought this was the component's changelog. You can ignore my comment 😇

**Pull request merged**